src/ck-ldb/LBComm.C

```diff
@@ -139,7 +140,7 @@ int LBCommData::compute_key()
 
   int k=-1;
   for(int i=0; i < (pcount+3)/4; i++)
-    k ^= kstring[i];
+    k ^= kintarr[i];
```

Comment:
Nice bugfix. While you're here, the loop should probably compare with `(pcount + (sizeof(int)-1))/sizeof(int)`